aiida_codtools/cli/misc/cif_import.py

```diff
@@ -156,10 +160,13 @@ def launch(group, database, max_entries, number_species, skip_partial_occupancie
                 if skip_partial_occupancies and cif.has_partial_occupancies():
                     click.echo('Cif<{}> skipped: contains partial occupancies'.format(source_id))
                 else:
-                    cif.store()
-                    group.add_nodes([cif])
+                    if not dry_run:
+                        cif.store()
+                        group.add_nodes([cif])
+                        click.echo('Cif<{}> added: new CifData<{}> to group {}'.format(source_id, cif.pk, group.name))
+                    else:
+                        click.echo('Cif<{}> would have added: CifData<{}> to group {}'.format(source_id, cif.uuid, group.name))
```

Comment:
what about printing the db_name here as well (COD, ICSD, etc.)? same on line 166